### PR TITLE
Add: Hero.jsx

### PR DESCRIPTION
**Pull request by gauravsinhaweb**
### Files to check:

- hero.jsx
- routes.jsx
- navbar.jsx

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  font-size: 1.4rem;
+  line-height: 2rem;
+}
+/* loader  */
+nav {
+  backdrop-filter: blur(16px);
+  border-radius: 0px 0px 8px 8px;
+}
+
+#spinnerBody {
```
Comment (bhtibrewal) [inline src/App.css:24]: Suggestion: maintain consistently with CSS selectors

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  background-size: cover;
+  overflow: hidden;
+}
+.h-340 {
+  height: 34rem;
+}
+.text-1xl {
+  font-size: 1.4rem;
+  line-height: 2rem;
+}
```
Comment (PuneetP16) [inline src/App.css:17]: Suggestion: Give proper css formating.

File: src/App.css (hunk 1 above)
Comment (PuneetP16) [inline src/App.css:24]: Suggestion: Either use ID or classes only for using styles for consistency

File: src/routes/RoutePage.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { Navbar } from "../components/index-component";
+import { Hero, Quiz, Rules } from "../pages/pages-index";
+import { ScrollToTop } from "../services/ScrollToTop";
+
+function RoutePage() {
+  // const { loading, setLoading } = useState();
```
Comment (PuneetP16) [inline src/routes/RoutePage.jsx:9]: Remove commented code.

File: src/routes/RoutePage.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            <div className="sk-cube-grid ">
+              <div className="sk-cube sk-cube1"></div>
+              <div className="sk-cube sk-cube2"></div>
+              <div className="sk-cube sk-cube3"></div>
+              <div className="sk-cube sk-cube4"></div>
+              <div className="sk-cube sk-cube5"></div>
+              <div className="sk-cube sk-cube6"></div>
+              <div className="sk-cube sk-cube7"></div>
+              <div className="sk-cube sk-cube8"></div>
+              <div className="sk-cube sk-cube9"></div>
```
Comment (PuneetP16) [inline src/routes/RoutePage.jsx:32]: Here too

File: src/utils/functions/Timer.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from "react";
+import { useDataContext } from "../../context";
+
+export function Timer() {
+  const { isOpen } = useDataContext();
+  const [time, setTime] = useState({
+    minute: "15",
+    second: "59",
+  });
```
Comment (PuneetP16) [inline src/utils/functions/Timer.js:9]: Suggestion: use numbers instead of strings for the time

**Review by PuneetP16 — APPROVED** [review]
Appreciation: UI is awesome, added some suggestions regarding refactoring.

File: src/backend/controllers/HistoryController.js [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+ * All the routes related to User History are present here.
+ * These are private routes.
+ * Client needs to add "authorization" header with JWT token in it to access it.
+ * */
+
+/**
+ * This handler handles getting videos from user's history.
+ * send GET Request at /api/user/history
+ * */
+export const getHistoryVideosHandler = function (schema, request) {
```
Comment (bhtibrewal) [inline src/backend/controllers/HistoryController.js]: Remove these history, playlist and likes controllers

**Review by bhtibrewal — APPROVED** [review]
Written few suggestions you can look into them

File: src/server.js [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+} from "./backend/controllers/WatchLaterController";
+export function makeServer({ environment = "development" } = {}) {
+  return new Server({
+    serializers: {
+      application: RestSerializer,
+    },
+    environment,
+    // TODO: Use Relationships to have named relational Data
+    models: {
+      video: Model,
```
Comment (Shivani-1524) [inline src/server.js]: suggestion: remove the unnecessary backend code

File: src/utils/apiCall/getData.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+export const getData = async (request, url, dispatch) => {
+  try {
+    const res = await axios({
+      method: request,
+      url: url,
+    });
+    if (res.status === 200) {
```
Comment (Shivani-1524) [inline src/utils/apiCall/getData.js]: suggestion: check for both status 200 and 201 as mockbee uses them interchangeably.

File: src/components/navbar/Navbar.jsx [hunk 9]
```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { FaUserCircle } from "react-icons/fa";
+import { GiHamburgerMenu } from "react-icons/gi";
+import {
```
Comment (Shivani-1524) [inline src/components/navbar/Navbar.jsx:2]: export and import icons from an assets/icon.js folder and use it across your app

**Review by Shivani-1524 — APPROVED** [review]
Code looks good to merge

**Pull request merged**